source/install.txt

```diff
@@ -125,11 +140,6 @@ Procedure
       
             .. include:: /includes/steps/install-shell-linux-generic.rst
 
-.. important:: Windows Users
```

Comment:
Hey @rose-m , I noticed we were accidentally documenting the default preference directory for Windows twice, but listing two different locations:

- ``C:\Users\<username>'\.mongodb\mongosh`` and
- ``C:\Users\<username>\AppData\Roaming``

Can you confirm which directory is correct? I _think_ it's the former, but want to verify.

Reply:
So for Windows we try to resolve to `%APPDATA%` and use both `Roaming` and `Local`. In `%APPDATA%\Roaming` we store files like the configuration options (enable telemetry etc.) and in `%APPDATA%\Local` we store our log files for example.
The old shell used `.mongodb\mongosh` and we still use that on *nix platforms but not for Windows :)

Reply:
I fixed the config / preference option directory to be `%APPDATA%/mongodb/mongosh`, since when I just tested on Windows 10, `%APPDATA%` goes to `AppData/Roaming`.

I also updated the Windows directory on the [Logs Page](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-12464/logs/) to be `%UserProfile%/AppData/Local/mongodb/mongosh`